src/Jackett.Common/Indexers/Definitions/YggTorrent.cs

```diff
@@ -0,0 +1,1063 @@
+
+        // Regex for isolated episode numbers (not preceded by S or E)
+        private static readonly Regex _StandaloneEpisode3Digits = new Regex(
+            @"(?<![SE\d])(?<=[\s\.\-\[\(]|^)(\d{3})(?=[\s\.\-\]\)]|$)",
+            RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+        private static readonly Regex _StandaloneEpisode2Digits = new Regex(
+            @"(?<![SE\d])(?<=[\s\.\-\[\(]|^)(\d{2})(?=[\s\.\-\]\)]|$)",
+            RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+        private static readonly Regex _StandaloneEpisode4Digits = new Regex(
+            @"(?<![SE\d])(?<=[\s\.\-\[\(]|^)(\d{4})(?=[\s\.\-\]\)]|$)",
+            RegexOptions.Compiled | RegexOptions.IgnoreCase);
+
+        #endregion
+
+        #region Configuration Keys
+
+        private const string CfgUsername = "username";
+        private const string CfgPassword = "password";
+        private const string CfgMultiLang = "multilang";
+        private const string CfgMultiLanguageValue = "multilanguage";
+        private const string CfgVostfr = "vostfr";
+        private const string CfgFilterTitle = "filter_title";
+        private const string CfgStripSeason = "strip_season";
+        private const string CfgEnhancedAnime = "enhancedAnime";
+        private const string CfgEnhancedAnime4 = "enhancedAnime4";
+        private const string CfgSort = "sort";
+        private const string CfgOrder = "type";
+        private const string CfgTurboAccount = "turbo_account";
+
+        #endregion
+
+        // Cache for configuration options (prevents repeated access)
+        private bool? _cachedTurboAccount;
+        private bool? _cachedMultiLang;
+        private string _cachedMultiLangValue;
+        private bool? _cachedVostfr;
+        private bool? _cachedFilterTitle;
+        private bool? _cachedStripSeason;
+        private bool? _cachedEnhancedAnime;
+        private bool? _cachedEnhancedAnime4;
+
+        public YggTorrent(
+            IIndexerConfigurationService configService,
+            WebClient wc,
+            Logger l,
+            IProtectionService ps,
+            ICacheService cs)
+            : base(
+                configService: configService,
+                client: wc,
+                logger: l,
+                p: ps,
+                cacheService: cs,
+                configData: new ConfigurationData())
+        {
+            InitializeConfiguration();
+        }
+
+        private void InitializeConfiguration()
+        {
+            // Authentification
+            configData.AddDynamic(CfgUsername, new StringConfigurationItem("Username") { Value = "" });
+            configData.AddDynamic(CfgPassword, new PasswordConfigurationItem("Password") { Value = "" });
+
+            // Turbo Account - skip the 30-second delay
+            configData.AddDynamic(
+                CfgTurboAccount,
+                new BoolConfigurationItem("I have a Turbo account (skips 30s download delay)")
```

Comment:
```suggestion
                new BoolConfigurationItem("I have a Turbo account")
```

Avoid any confusion for users without a Turbo account, as they might try ticking it thinking it will bypass the delay.